cypress/page/payment-step.page.ts

```diff
@@ -0,0 +1,26 @@
+class PaymentStepPage{
+    private buton: string;
+    private summary: string;
+    private text: string;
+
+    constructor(){
+        this.buton = ".bankwire";
+        this.summary = ".cart_navigation span";
+        this.text = "#center_column > div > p > strong";
+    }
+
+    public payByBankWire(): void {
+        cy.get(this.buton).click();
+    }
+
+    public confirmOrder(): void {
+        cy.get(this.summary).click();
+    }
+
+    public verifyText(): void {
+        cy.get(this.text).should("have.text",
```

Comment:
So yeah, just move this `should()` to the test itself, so one knows what is being tested without having to dig through POMs. There may be some cases where this approach is fine, but usually I'd prefer the former.

Reply:
Thank you for the advices, Santiago! I'm on it :)